examples/custom/book.js

```diff
@@ -1,13 +1,13 @@
 const fs = require("fs");
 module.exports = {
     "gitbook": "3.x.x",
-    "title": "gitbook-plugin-include-codeblock example",
+    "title": "gitbook-plugin-include-codeblock example-custom",
     "plugins": [
         "include-codeblock"
     ],
     "pluginsConfig": {
         "include-codeblock": {
-            "template": __dirname + "/user-template.hbs"
+            "template": __dirname + "user-template.hbs"
```

Comment:
It will be better that use `path.join(__dirname, "user-template.hbs")`.
(and need `const path = require("path")`)